nirc_ehr/src/org/labkey/nirc_ehr/dataentry/form/NIRCDeathNecropsyFormType.java

```diff
@@ -0,0 +1,43 @@
+package org.labkey.nirc_ehr.dataentry.form;
+
+import org.labkey.api.ehr.dataentry.DataEntryFormContext;
+import org.labkey.api.ehr.dataentry.FormSection;
+import org.labkey.api.ehr.dataentry.TaskForm;
+import org.labkey.api.module.Module;
+import org.labkey.api.view.template.ClientDependency;
+import org.labkey.nirc_ehr.dataentry.section.NIRCAnimalDetailsFormSection;
+import org.labkey.nirc_ehr.dataentry.section.NIRCDeathFormSection;
+import org.labkey.nirc_ehr.dataentry.section.NIRCDeathNecropsyWeightFormSection;
+import org.labkey.nirc_ehr.dataentry.section.NIRCGrossPathologyFormSection;
+import org.labkey.nirc_ehr.dataentry.section.NIRCNecropsyFormSection;
+import org.labkey.nirc_ehr.dataentry.section.NIRCTaskFormSection;
+import org.labkey.nirc_ehr.dataentry.section.NIRCTissueDispositionFormSection;
+
+import java.util.Arrays;
+
+public class NIRCDeathNecropsyFormType extends TaskForm
```

Comment:
performedBy value isn't sticking for me in the various sections of this form

Reply:
fixed